app/screens/select_server/select_server.js

```diff
@@ -85,10 +83,9 @@ class SelectServer extends PureComponent {
     }
 
     handleLoginOptions = () => {
-        const {config, intl, license, serverVersion, theme} = this.props;
-        const version = serverVersion.match(/^[0-9]*.[0-9]*.[0-9]*(-[a-zA-Z0-9.-]*)?/g)[0];
```

Comment:
This'll fix https://mattermost.atlassian.net/browse/RN-395, but we might want to understand why some servers aren't sending back a version for that ticket

Reply:
This was being cause by https://mattermost.zendesk.com/agent/tickets/4809

Reply:
Ah okay, I'll link to that on Jira and resolve the issue in Sentry